src/NuGet.Core/NuGet.Commands/PushRunner.cs

```diff
@@ -50,6 +50,7 @@ public static class PushRunner
                 timeoutSeconds,
                 disableBuffering,
                 endpoint => CommandRunnerUtility.GetApiKey(settings, endpoint, apiKey),
+                symbolsEndpoint => CommandRunnerUtility.GetApiKey(settings, symbolsEndpoint, symbolsApiKey),
```

Comment:
Maybe... if you use "nuget.exe setapikey XXX" then it'll set the same API key for both nuget.org and nuget.smbsrc.net. So people who use that won't be broken.

What I can do to be safe is use symbolsApiKey if it's set, otherwise use apiKey. That would be for the third parameter to GetApiKey() here

---
In reply to: [68441601](https://github.com/NuGet/NuGet.Client/pull/712#discussion_r68441601) [](ancestors = 68441601)